modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -74,6 +79,25 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
     assertShadowHostState(editor, shouldExist);
   };
 
+  const createNotification = (editor: Editor) =>
```

Comment:
The function name seems misleading. It opens a dialog instead of creating one right?

Reply:
This creates a notification, not a dialog.

Reply:
Sorry. I meant notification :)

Reply:
Is there much distinction between creating and opening a notification? Docs calls this `Create custom notification`